runtime/src/test/scala/akka/grpc/GrpcClientSettingsSpec.scala

```diff
@@ -153,10 +155,12 @@ class GrpcClientSettingsSpec extends WordSpec with Matchers with ScalaFutures {
     }
 
     "fail to parse configuration with non-existent certificate" in {
+      val system = sysWithCert("no-such-cert.pem")
       val thrown = the[IllegalArgumentException] thrownBy
-        GrpcClientSettings.fromConfig("project.WithSpecificConfiguration")(sysWithCert("no-such-cert.pem"))
+        GrpcClientSettings.fromConfig("project.WithSpecificConfiguration")(system)
       // We want a good message since missing classpath resources are difficult to debug
       thrown.getMessage should include("certs/no-such-cert.pem")
+      system.terminate()
```

Comment:
try-finally because now if it fails it leaks the system, also `TestKit.shutdownActorSystem`